src/aliases/service/aliases/delete.go

```diff
@@ -0,0 +1,27 @@
+package aliases
+
+import (
+	"context"
+
+	"github.com/consensys/quorum-key-manager/pkg/errors"
+	auth "github.com/consensys/quorum-key-manager/src/auth/entities"
+)
+
+func (s *Aliases) Delete(ctx context.Context, registry, key string, userInfo *auth.UserInfo) error {
+	logger := s.logger.With("registry", registry, "key", key)
+
+	_, err := s.Get(ctx, registry, key, userInfo)
```

Comment:
yes because this is what checks the access to the resource. By getting the alias, you verify that the "tenant" is part of the "allowed_tenants" of the registry. Even if we do not use the returned alias, we check the error and will return 404 if the user is not allowed.